x/evm/module.go

```diff
@@ -208,6 +208,10 @@ func (am AppModule) RegisterServices(cfg module.Configurator) {
 	_ = cfg.RegisterMigration(types.ModuleName, 10, func(ctx sdk.Context) error {
 		return migrations.MigrateCastAddressBalances(ctx, am.keeper)
 	})
+
+	_ = cfg.RegisterMigration(types.ModuleName, 11, func(ctx sdk.Context) error {
+		return migrations.AddNewParamsAndSetAllToDefaults(ctx, am.keeper)
```

Comment:
lets not use this migration handler because it will reset params which we don't want to do, instead we should initialize the param value for the new param being introduced. And lets add tests for that

Reply:
Yeah, that's a valid concern, let's only change for the new param